libobs/audio-monitoring/pulse/pulseaudio-wrapper.c

```diff
@@ -61,28 +61,43 @@ void get_default_id(char **id)
 	pulseaudio_unref();
 }
 
+/**
+ * Checks whether a the sound source (id1) is the .monitor device for the
+ *	selected monitoring output (id2).
+ */
 bool devices_match(const char *id1, const char *id2)
 {
 	bool match;
+	char *name_default = NULL;
 	char *name1 = NULL;
 	char *name2 = NULL;
 
+	get_default_id(&name_default);
```

Comment:
Move this after the early return so that it wont leak the allocated string.

Reply:
Good catch. Fixed.